mkdocs.yml

```diff
@@ -110,18 +111,23 @@ plugins:
             - https://distributed.dask.org/en/latest/objects.inv
           paths: [ src ]  # search packages in the src folder
           options:
+            heading_level: 1
             inherited_members: true
             docstring_style: google
             docstring_section_style: spacy
             line_length: 80
             show_bases: true
             members_order: source
             show_submodules: false
-            show_signature_annotations: false
+            separate_signature: true
+            show_signature_annotations: true
```

Comment:
Changing these options (or at least one of them) affects the highlighting and readability in the API docs.

| Develop | Current Branch | Suggestion |
|---|---|---|
| ![image](https://github.com/aai-institute/pyDVL/assets/27914730/1ed732d5-7433-465c-82cf-516c0a9090cd) | ![image](https://github.com/aai-institute/pyDVL/assets/27914730/b95aaa18-8751-40e8-868e-7235a47327f9) | ![image](https://github.com/aai-institute/pyDVL/assets/27914730/a9191d07-4b41-4325-87bf-633da080b560) |

Here's the suggestion extra css used for this:

```css
/* Highlight function names in API docs */
span.doc-function-name {
    color: var(--doc-symbol-function-fg-color);
    background-color: var(--md-code-bg-color);
    font-weight: 500;
}

/* Highlight class names in API docs */
span.doc-class-name {
    color: var(--doc-symbol-class-fg-color);
    background-color: var(--md-code-bg-color);
    font-weight: 500;
}

/* Highlight attribute names in API docs */
span.doc-attribute-name {
    color: var(--doc-symbol-attribute-fg-color);
    background-color: var(--md-code-bg-color);
    font-weight: 500;
}
```

Reply:
Hmm... I actually like the muted style. I tried to make the headings with more prominent with [show_symbol_type_heading: true](https://mkdocstrings.github.io/python/usage/configuration/headings/#show_symbol_type_heading) but for some reason neither that nor `show_symbol_type_toc` work.

I'm not a huge fan of that many colors, but if your changes are compatible with dark mode and they include all symbol types we can include them if @schroedk agrees

Reply:
Scratch that. After updating with the versions you pinned, the symbol headings appear 🎉

Reply:
Yes, this looks much nicer!